app/controllers/users_controller.rb

```diff
@@ -164,7 +164,6 @@ def onboarding_update
 
   def onboarding_checkbox_update
     # TODO: mstruve will remove once debugging is done
```

Comment:
```suggestion

```
Since you added a reminder `TODO` on line 179, probably could remove this one?

Reply:
I figured a TODO for each set of lines to make sure I dont forget any. Better safe than sorry 🤷‍♀️

Reply:
Whatever works best to remind you works for me! 👍🏽